examples/HastyNoise/config.cmake

```diff
@@ -0,0 +1,8 @@
+hunter_config(Boost VERSION ${HUNTER_Boost_VERSION}
+    CMAKE_ARGS
+        BUILD_SHARED_LIBS=ON 
```

Comment:
I think we only need `BUILD_SHARED_LIBS=ON` here, so it will be easier to use `HUNTER_BUILD_SHARED_LIBS=ON` instead:
- https://github.com/ruslo/hunter/blob/v0.23.203/examples/Boost-python/CMakeLists.txt#L6

Reply:
checking if that will work

https://travis-ci.org/caseymcc/hunter/builds/556930483
https://ci.appveyor.com/project/caseymcc/hunter/builds/25885544

Reply:
looks good